File: Gaming-Wiki/GamingWikiServer/GameTable.java
import java.sql.*;  
public class GameTable
{
    public DB db;
    public GameTable()
    {
        this.db = new DB();
    }

    public static void main(String args[]) {
        GameTable gameTable = new GameTable();
        
    }
    
    public void getAll() {
        try {
            Statement stmt = this.db.conn.createStatement();
            ResultSet rs = stmt.executeQuery( "SELECT * FROM GameTable" );
        } catch(Exception e) {
            System.out.println("Something went wrong in game Table");
        }
    }
}

File: Gaming-Wiki/Server/Services/game-table.php
<?php

/**
 * Service for dealing with the Game table
 */
class GameTable {
    private $db;
    function __construct($d) {
        $this->db = $d;
    }

    /**
     * SQL for inserting an Game into the DB
     */
    public function insert($gameName, $description) {
        $query = "INSERT INTO GameTable (GameName, Description)
                  VALUES ('$gameName', '$description')";
        $this->db->ExecuteNonQuery($query);
    }
	
	public function RemoveGame($gameName) {
		
    $query =  "DELETE FROM GameTable WHERE GameName = '$gameName' ";
    $this->db->ExecuteNonQuery($query);
  
 }

    /**
     * SQL for Selecting a single Game record from the DB
     */
    public function get($gameName) {
        $query = "SELECT * FROM GameTable WHERE GameName = '$gameName' ";
        $response = $this->db->ExecuteQuery($query);
        return $response;
    }

    /**
     * SQL for getting all games from the database
     */
    public function getAll() {
        $query = "SELECT GameName FROM  GameTable";
        $response = $this->db->ExecuteQuery($query);
        return $response;
    }
}

?>